### PR TITLE
feat: add zh_CN translation for frontend

### PR DESCRIPTION
**Pull request by genment**

### CODE REVIEW

**Comment by robinv8** [comment]
@genment There are a few things that you need to modify

**Comment by genment** [comment]
Hi, I just push a new commit.

P.S.
Those not-in-use text are not translated because I can't translate without a context.

**Comment by robinv8** [comment]
> Hi, I just push a new commit.
> 
> P.S. Those not-in-use text are not translated because I can't translate without a context.

**Comment by robinv8** [comment]
> 



> Hi, I just push a new commit.
> 
> P.S. Those not-in-use text are not translated because I can't translate without a context.

There are still some places that need to be modified

**Comment by genment** [comment]
Could you specify?

**Review by robinv8 — APPROVED** [review]

File: ui/src/components/Operate/index.tsx [hunk 1]
```diff
@@ -125,7 +125,7 @@ const Index: FC<IProps> = ({
               to={editUrl}
               className="link-secondary p-0 fs-14 me-3"
               style={{ lineHeight: '23px' }}>
-              {item.name}
+              {t(`btn_${item.action}`)}
```
[the remark this answers is not in the document]
Comment (Jeremyrennerofficial) [reply ui/src/components/Operate/index.tsx]: Perfectly ok****_[image]()_****

**Pull request merged**